sdk/rust/examples/backend/src/configuration.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(clap::Parser)]
+#[command(version, about)]
+pub struct Configuration {
```

Comment:
This doesn't look to be used? 

if you want env here as well. You can enable the env flag for `clap`, then it can take an env = "PORT" as a backup if the flag isn't set on the cli.